src/app/pages/k-12/k-12.js

```diff
@@ -30,9 +30,9 @@ export default class K12 extends BaseView {
             bucketClass: 'allies',
             hasImage: false,
             titleText: 'OpenStax Allies',
-            blurbHtml: `OpenStax allies provide additional tools alongside our
-                OpenStax texts, because we believe that education is a community
-                effort.`,
+            blurbHtml: `We partner with OpenStax Allies to provide additional integrated
+            resources for our AP® course textbooks. Discover the courseware, online homework, 
```

Comment:
Missed that in the copy. Good eye. Fixed.

On 03/30/2016 12:13 PM, Roy Johnson wrote:
>
> In src/app/pages/k-12/k-12.js 
> <https://github.com/openstax/os-webview/pull/163#discussion_r57926544>:
>
> > @@ -30,9 +30,9 @@ export default class K12 extends BaseView {
> >              bucketClass: 'allies',
> >              hasImage: false,
> >              titleText: 'OpenStax Allies',
> > -            blurbHtml: `OpenStax allies provide additional tools alongside our
> > -                OpenStax texts, because we believe that education is a community
> > -                effort.`,
> > +            blurbHtml: `We partner with OpenStax Allies to provide additional integrated
> > +            resources for our AP® course textbooks. Discover the courseware, online homework,
>
> Should use |&reg;| instead of |®| within HTML.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/openstax/os-webview/pull/163/files/31d127ce5852c785ae2a48aaa5c321a1da96cc7c#r57926544>
>
> !DSPAM:1106,56fc092f149562101914662!